src/modelconfig.cpp

```diff
@@ -504,7 +503,12 @@ Status ModelConfig::parseModelMapping() {
 
 Status ModelConfig::parseNode(const rapidjson::Value& v) {
     this->setName(v["name"].GetString());
-    this->setBasePath(v["base_path"].GetString());
+    try {
+        this->setBasePath(v["base_path"].GetString());
+    } catch (std::logic_error& e) {
+        SPDLOG_DEBUG("Relative path error: {}", e.what());
+        return StatusCode::INTERNAL_ERROR;
```

Comment:
This is internal error since we are already after schema validation?

Reply:
Yes.